Runtime/JSAsyncScope.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Threading;
+
+namespace NodeApi
+{
+    /// <summary
+    /// Keeps JSSynchronizationContext queue alive until the end of the 
+    public class JSAsyncScope : IDisposable
```

Comment:
Missing end of sentence and `</summary>`.

Reply:
More complete comments are written,